ui/src/pages/Upload.tsx

```diff
@@ -112,149 +176,206 @@ export const Upload: React.FC = () => {
-            <Select
-              value={formInput.year}
+              <StyledMenuItem value={"Toronto"}>Toronto</StyledMenuItem>
+              <StyledMenuItem value={"Calgary"}>Calgary</StyledMenuItem>
+              <StyledMenuItem value={"Vancouver"}>Vancouver</StyledMenuItem>
+              <StyledMenuItem value={"Thunder Bay"}>Thunder Bay</StyledMenuItem>
+            </StyledSelect>
+            <FormControl>
+              <UploadLabelsText>Year Published</UploadLabelsText>
+              <StyledSelect
+                variant="outlined"
+                value={formInput.year}
+                onChange={handleChange}
+                defaultValue={"2021"}
+                inputProps={{
+                  name: "year",
+                  id: "select-label-year",
+                }}
+              >
+                <StyledMenuItem value={"2021"}>2021</StyledMenuItem>
+                <StyledMenuItem value={"2020"}>2020</StyledMenuItem>
+                <StyledMenuItem value={"2019"}>2019</StyledMenuItem>
+              </StyledSelect>
+            </FormControl>
+            <FormControl>
+              <UploadLabelsText>Tags</UploadLabelsText>
+              <StyledTags
+                autoHighlight
+                multiple
+                id="tags-outlined"
+                name="tags"
+                freeSolo
+                options={tagOptions ? tagOptions : [""]}
+                filterSelectedOptions
+                onChange={(_, newValue) => setTagArrayValues(newValue)}
+                value={tagArray ? tagArray : [""]}
+                renderTags={(value, getTagProps) =>
+                  value.map((option, index) => (
+                    <Chip
+                      key={option}
+                      variant="outlined"
+                      label={option}
+                      {...getTagProps({ index })}
+                    />
+                  ))
+                }
+                renderInput={(params: TagParameters) => {
+                  params.inputProps.onKeyDown = handleKeyDown;
+                  return (
+                    <TextField
+                      {...params}
+                      variant="outlined"
+                      label="Shoe Tags"
+                      placeholder="Select Tags"
+                    ></TextField>
+                  );
+                }}
+              />
+            </FormControl>
+          </StyledBackgroundColor>
+          <StyledBackgroundColor>
+            <UploadStoriesHeading>Multimedia</UploadStoriesHeading>
+            <ImageContainer>
+              <UploadLabelsText>Add Image</UploadLabelsText>
+              <StyledDropzoneArea
+                showFileNames
+                acceptedFiles={["image/*"]}
+                filesLimit={1}
+                dropzoneText={"Drag image here or select from device"}
+                onChange={(files) => {
+                  setImage(files);
+                }}
```

Comment:
Does adding an image show a preview on the screen, cuz i think design on Figma has that?

https://www.figma.com/file/uVqnweL2ApqvPHWzQYmZ1t/Dashboard?node-id=633%3A381

Reply:
yup it does, I know it's not exactly the same styling as the figma though but we can fix that later on.
![image](https://user-images.githubusercontent.com/19617248/109397548-2ce5ff00-7905-11eb-8457-847e21dc2051.png)